builder/bundles.go

```diff
@@ -480,7 +506,35 @@ func downloadRpms(packagerCmd, rpmList []string, baseDir string, maxRetries int)
 	return nil, downloadErr
 }
 
-func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *bundle, downloadRetries int) error {
+func extractRpm(baseDir string, rpm string) error {
+	cmd := exec.Command("rpm2cpio", rpm)
+	cmd.Env = os.Environ()
+	var outBuf bytes.Buffer
+	cmd.Stdout = &outBuf
+
+	err := cmd.Run()
+	if err != nil {
+		fmt.Println("cmd1 failed")
```

Comment:
This isn't a very actionable error. I assume that the err being returned has the command being run and the result in it so maybe just return instead. Otherwise rpm2cpio failed would be more useful.